spec/features/users/login_spec.rb

```diff
@@ -93,4 +93,45 @@
       expect(page).to have_content("Welcome, #{merchant.name}, you are logged in!")
     end
   end
+  describe "all users can log out" do
+      it 'can log out as a registered user' do
+      user = User.create(name: "Jordan",
+                          address: "394 High St",
+                          city: "Denver",
+                          state: "CO",
+                          zip_code: "80602",
+                          email: "hotones@hotmail.com",
+                          password: "password",
+                          password_confirmation: "password",
+                          role: 0)
+
+      visit '/'
+
+      click_link "Login"
+      expect(current_path).to eq('/login')
+
+      fill_in :email, with: user.email
+      fill_in :password, with: user.password
+
+      click_button "Login"
+
+      expect(current_path).to eq('/profile')
+      expect(page).to have_content("Welcome, #{user.name}, you are logged in!")
+
+
+      within "nav" do
+        expect(page).not_to have_link("Login")
+        expect(page).not_to have_link("Register")
+        expect(page).to have_link("Log Out")
+      end
+      @meg = Merchant.create(name: "Meg's Bike Shop", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80203)
```

Comment:
We don't need lines 127-130, we're not testing cart functionality right now.

Reply:
I think what he was doing was adding stuff to the cart to make sure it is empty after?